solr-8/Dockerfile

```diff
@@ -0,0 +1,22 @@
+FROM solr:8
+
+EXPOSE 8983
+
+# TODO: Update with master / new release branch when merged
+ARG CKAN_BRANCH="solr-8.0"
+
+ENV SOLR_CONFIG_DIR="/opt/solr/server/solr/configsets"
+ENV SOLR_SCHEMA_FILE="$SOLR_CONFIG_DIR/ckan/conf/managed-schema"
+
+USER root
+
+# Create a CKAN configset by copying the default one
+RUN cp -R $SOLR_CONFIG_DIR/_default $SOLR_CONFIG_DIR/ckan
+
+# Update the schema
+ADD https://raw.githubusercontent.com/ckan/ckan/$CKAN_BRANCH/ckan/config/solr/schema.xml $SOLR_SCHEMA_FILE
+RUN chmod 644 $SOLR_SCHEMA_FILE 
+
+USER solr
+
+CMD ["sh", "-c", "solr-precreate ckan $SOLR_CONFIG_DIR/ckan"]
```

Comment:
+1 on naming the core ckan (instead of master) so it matches the default in CKAN config files.